src/components/Editor.js

```diff
@@ -24,124 +24,159 @@ class Editor extends Component {
-          { this.path('parity', 'keys_path', base, platform) }
-          { this.text('parity', 'identity') }
+        <Section title={data.core.section} description={data.core.description}>
+          { this.text('core', 'alertnotify') }
+          { this.text('core', 'blocknotify') }
+          { this.flag('core', 'blocksonly') }
+          { this.text('core', 'assumevalid') }
+          { this.path('core', 'conf', base, platform) }
+          { this.flag('core', 'daemon') }
+          { this.path('core', 'datadir', base, platform) }
+          { this.number('core', 'dbcache') }
+          { this.flag('core', 'feefilter') }
+          { this.text('core', 'loadblock') }
+          { this.number('core', 'maxorphantx') }
+          { this.number('core', 'maxmempool') }
+          { this.number('core', 'mempoolexpiry') }
+          { this.number('core', 'blockreconstructionextratxn') }
+          { this.number('core', 'par') }
+          { this.text('core', 'pid') }
+          { this.number('core', 'prune') }
+          { this.flag('core', 'reindex-chainstate') }
+          { this.flag('core', 'reindex') }
+          { this.flag('core', 'sysperms') }
+          { this.flag('core', 'txindex') }
         </Section>
-        <Section title={data.account.section} description={data.account.description}>
-          { this.list('account', 'unlock') }
-          { this.text('account', 'password') }
-          { this.number('account', 'keys_iterations') }
+        <Section title={data.debug.section} description={data.debug.description}>
+          { this.text('debug', 'uacomment') }
+          { this.number('debug', 'checkblocks') }
+          { this.number('debug', 'checklevel') }
+          { this.number('debug', 'checkmempool') }
+          { this.flag('debug', 'checkpoints') }
+          { this.flag('debug', 'disablesafemode') }
+          { this.flag('debug', 'testsafemode') }
+          { this.number('debug', 'dropmessagestest') }
+          { this.number('debug', 'fuzzmessagestest') }
+          { this.flag('debug', 'stopafterblockimport') }
+          { this.number('debug', 'limitancestorcount') }
+          { this.number('debug', 'limitancestorsize') }
+          { this.number('debug', 'limitdescendantcount') }
+          { this.number('debug', 'limitdescendantsize') }
+          { this.text('debug', 'bip9params') }
+          { this.select('debug', 'debug') }
+          { this.flag('debug', 'logips') }
+          { this.flag('debug', 'logtimestamps') }
+          { this.flag('debug', 'logtimemicros') }
+          { this.number('debug', 'mocktime') }
+          { this.number('debug', 'maxsigcachesize') }
+          { this.number('debug', 'maxtipage') }
+          { this.decimal('debug', 'maxtxfee') }
+          { this.flag('debug', 'printtoconsole') }
+          { this.text('debug', 'promiscuousmempoolflags') }
+          { this.flag('debug', 'shrinkdebugfile') }
+          { this.flag('debug', 'testnet') }
```

Comment:
`testnet` and `regtest` should not be individual boolean options in the `debug` section, but should be a `.select` style option in the `core` section where they user can select [mainnet, testnet3, regtest, simnet] with mainnet (no option) as the default.